src/components/AppHeader/AppHeader.css

```diff
@@ -2,6 +2,8 @@
 
 .appBar {
   background-color: var(--backgroundAltColor);
+  height: auto;
+  padding: 8px 2.4rem;
```

Comment:
I'd like to see us stick to pixels for units, but open to discussion on it if you feel strongly.